Image Feed/ImagesList/ImagesListViewController.swift

```diff
@@ -11,6 +13,18 @@ class ImagesListViewController: UIViewController {
         photosName = Array(0..<20).map{ "\($0)" }
     }
 
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if segue.identifier == ShowSingleImageSegueIdentifier {
+            let viewController = segue.destination as! SingleImageViewController
```

Comment:
В целом, для обработки UIStoryboardSegue часто форсом кастят в продакшене, однако я бы не рекомендовала так делать в проектах для портфолио, лучше скастить в ифе

```suggestion
        if segue.identifier == ShowSingleImageSegueIdentifier,
            let viewController = segue.destination as? SingleImageViewController {
            
```

Reply:
аналогично для других форс-кастов ниже